crates/rpc/types/src/types.rs

```diff
@@ -91,7 +91,7 @@ impl RpcL1Status {
             last_rpc_error: l1s.last_rpc_error,
             cur_height: l1s.cur_height,
             cur_tip_blkid: l1s.cur_tip_blkid,
-            last_published_txid: l1s.last_published_txid,
+            last_published_txid: l1s.last_published_txid.map(Into::into),
```

Comment:
Does this work?

```suggestion
            last_published_txid: l1s.last_published_txid.into(),
```

Reply:
Oh yes, it should have worked, but interestingly it did not work.